.github/workflows/ci.yaml

```diff
@@ -66,6 +66,24 @@ jobs:
           # return to previously saved path
           popd
         done
+    - name: Java Code Coverage
+      timeout-minutes: 20
+      run: |
+        set -x
```

Comment:
Thanks for finding this workaround!  Would it be possible to make the output more readable, currently prints ~80 lines so it is difficult to see where the coverage is

Reply:
+1

Reply:
Is it possible to prevent github actions from printing out the commands it is running? That seems to be most of it

Reply:
Whoops, my bad, this was just happening because I left the "set -x" in there!